tests/blob/Makefile

```diff
@@ -0,0 +1,10 @@
+DEVELHELP ?= 0
```

Comment:
Why disabling DEVELHELP by default? I would keep it.

Reply:
yup, removed the line (default is 1)